src/courseware/course/sidebar/sidebars/course-outline/SidebarUnit.jsx

```diff
@@ -18,7 +18,7 @@ const SidebarUnit = ({
   const {
     complete,
     title,
-    icon,
+    icon = UNIT_ICON_TYPES.other,
   } = unit;
 
```

Comment:
Add coverage for `const iconType = isLocked ? UNIT_ICON_TYPES.lock : icon;` after merge https://github.com/raccoongang/frontend-app-learning/pull/24